src/main/java/uk/gov/hmcts/reform/civil/model/ServedDocumentFiles.java

```diff
@@ -26,29 +26,41 @@
 
     @JsonIgnore
     public List<String> getErrors() {
+        List<String> errors = getErrorsAddOrAmendDocuments();
+
+        if (ofNullable(particularsOfClaimDocumentNew).isEmpty() && ofNullable(particularsOfClaimText).isEmpty()) {
+            errors.add("You must add Particulars of claim details");
```

Comment:
the errors here could be extracted out to be common, but not the end of the world